Failed: CircuitBreakerSpec

https://jenkins.akka.io:8498/job/akka-multi-node-nightly/5107/consoleFull

<details>
<pre>
[info]   - when exception is defined as Success *** FAILED *** (2 seconds, 23 milliseconds)
[info]     java.lang.AssertionError: assertion failed: timeout 2 seconds expired:
[info]     at scala.Predef$.assert(Predef.scala:170)
[info]     at akka.testkit.TestKitBase$class.poll$1(TestKit.scala:276)
[info]     at akka.testkit.TestKitBase$class.awaitCond(TestKit.scala:282)
[info]     at akka.testkit.TestKit.awaitCond(TestKit.scala:831)
[info]     at akka.pattern.CircuitBreakerSpec$$anonfun$14$$anonfun$apply$mcV$sp$42$$anonfun$apply$mcV$sp$43.apply$mcV$sp(CircuitBreakerSpec.scala:672)
[info]     at akka.pattern.CircuitBreakerSpec$$anonfun$14$$anonfun$apply$mcV$sp$42$$anonfun$apply$mcV$sp$43.apply(CircuitBreakerSpec.scala:668)
[info]     at akka.pattern.CircuitBreakerSpec$$anonfun$14$$anonfun$apply$mcV$sp$42$$anonfun$apply$mcV$sp$43.apply(CircuitBreakerSpec.scala:668)
[info]     at org.scalatest.OutcomeOf$class.outcomeOf(OutcomeOf.scala:85)
[info]     at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]     at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]     at org.scalatest.Transformer.apply(Transformer.scala:20)
...
</pre>
</details>

--- a/akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala
+++ b/akka-actor-tests/src/test/scala/akka/pattern/CircuitBreakerSpec.scala
@@ -666,10 +666,10 @@ class CircuitBreakerSpec extends AkkaSpec with BeforeAndAfter with MockitoSugar 
 
     "reset failure count after exception in call" when {
       "exception is defined as Success" in {
-        val breaker = CircuitBreakerSpec.multiFailureCb()
-        breaker().withCircuitBreaker(Future(sayHi))
-        for (n ← 1 to 4) breaker().withCircuitBreaker(Future(throwException))
-        awaitCond(breaker().currentFailureCount == 4, awaitTimeout)
+        val breaker: CircuitBreakerSpec.Breaker = CircuitBreakerSpec.multiFailureCb()
+
+        for (_ ← 1 to 4) breaker().withCircuitBreaker(Future(throwException))
+        awaitCond(breaker().currentFailureCount == 4, awaitTimeout, message = s"Current failure count: ${breaker().currentFailureCount}")
 
         val harmlessException = new TestException
         val harmlessExceptionAsSuccess: Try[String] ⇒ Boolean = {
@@ -691,7 +691,7 @@ class CircuitBreakerSpec extends AkkaSpec with BeforeAndAfter with MockitoSugar 
       })
       checkLatch(breaker.openLatch)
       breaker().currentFailureCount should ===(1)
-      // Since the timeout should have happend before the inner code finishes
+      // Since the timeout should have happened before the inner code finishes
       // we expect a timeout, not TestException
       intercept[TimeoutException] {
         Await.result(fut, awaitTimeout)

